src/odemis/model/_metadata.py

```diff
@@ -228,6 +228,10 @@
 MD_FM_IMAGING_RANGE = "FM imaging range"  # dict str → [float, float] defining the volume of the FM imaging area, along x, y and z axes.
 MD_FAV_FM_POS_ACTIVE = "Favourite FM position active"  # dict str->float representing the position required for FM imaging
 MD_FAV_SEM_POS_ACTIVE = "Favourite SEM position active"  # dict -> float representing the position required for SEM imaging
+MD_FAV_FIB_POS_ACTIVE = "Favourite FIB position active"  # dict -> float representing the position required for FIB imaging
+MD_FAV_MILL_POS_ACTIVE = "Favourite Milling position active"  # dict -> float representing the position required for milling
+# NOTE: the MILL rx is defined as the angle between the FIB and the sample stage, not the stage tilt like the other metadata positions.
```

Comment:
The milling angle is the angle measured in clockwise direction between the sample plane and FIB axis. Assuming sample plane axis is positive from left to right and FIB axis is positive from up to down.